internal/formats/common/cyclonedxhelpers/format.go

```diff
@@ -97,18 +105,63 @@ func toBomDescriptor(name, version string, srcMetadata source.Metadata) *cyclone
 	}
 }
 
+func lookupRelationship(ty artifact.RelationshipType) bool {
+	switch ty {
+	case artifact.OwnershipByFileOverlapRelationship:
```

Comment:
I'm not 100% sure that this relationship belongs here, but I could be wrong

Reply:
I can remove this one for now.